IDGPredict/FacetPredict.h

```diff
@@ -20,384 +20,82 @@
 #define FACET_PREDICT_H
 
 #ifdef HAVE_IDG
-
-#include "DS9FacetFile.h"
```

Comment:
Where did this go?

Reply:
Into the new FacetPredict.cc. FacetPredict.h doesn't need this include anymore.